src/redux/sagas/colony/colonyCreate.ts

```diff
@@ -439,12 +437,10 @@ function* colonyCreate({
       /*
        * Deploy OneTx
        */
+      const oneTxHash = getExtensionHash(Extension.OneTxPayment);
+      const oneTxVersion = yield call(getOneTxPaymentVersion);
       yield put(
-        transactionAddParams(deployOneTx.id, [
-          getExtensionHash(Extension.OneTxPayment),
-          /* @TODO: get latest version of OneTxPayment extn programatically */
-          LATEST_ONE_TX_PAYMENT_VERSION || 0,
-        ]),
+        transactionAddParams(deployOneTx.id, [oneTxHash, oneTxVersion || 0]),
```

Comment:
And if you default `getOneTxPaymentVersion` to `1` if no data is available from the query, than you can remove the fallback from here

```suggestion
        transactionAddParams(deployOneTx.id, [oneTxHash, oneTxVersion]),
```

Alternatively, if you don't want to do this change, make sure to default to `1`, not `0`, as `1` is the first version that can exist, `0` cannot